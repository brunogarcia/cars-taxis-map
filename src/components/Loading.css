.Loading-main {
    margin: 25% 0;
    padding: 2em;
}
