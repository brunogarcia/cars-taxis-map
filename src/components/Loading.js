import React from 'react';
import CircularProgress from '@material-ui/core/CircularProgress';
import { withStyles } from '@material-ui/core/styles';
import './Loading.css';

const styles = theme => ({
  progress: {
    margin: theme.spacing.unit * 2,
  },
});

const Loading = () => {
  const message = 'Loading...';

  return (
    <div className="Loading-main">
      <CircularProgress size={50} />
      <p>{message}</p>
    </div>
  );
};

export default withStyles(styles)(Loading);
