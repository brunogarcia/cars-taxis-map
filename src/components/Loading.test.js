import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './Loading';

test('renders correctly', () => {
  const component = renderer.create(<Loading />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
