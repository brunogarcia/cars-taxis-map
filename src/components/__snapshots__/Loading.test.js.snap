// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Loading-main"
>
  <div
    className="MuiCircularProgress-root-2 MuiCircularProgress-colorPrimary-5 MuiCircularProgress-indeterminate-4"
    role="progressbar"
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg-7"
      viewBox="0 0 50 50"
    >
      <circle
        className="MuiCircularProgress-circle-8 MuiCircularProgress-circleIndeterminate-10"
        cx={25}
        cy={25}
        fill="none"
        r={20}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
  <p>
    Loading...
  </p>
</div>
`;
